.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -92,6 +92,17 @@ fun BuildSteps.substDirOnWindows(os: Os) {
     }
 }
 
+fun BuildSteps.cleanUpPerformanceBuildDir(os: Os) {
+    if (os == Os.WINDOWS) {
+        script {
+            name = "CLEAN_UP_PERFORMANCE_BUILD_DIR"
+            executionMode = BuildStep.ExecutionMode.ALWAYS
+            scriptContent = """rmdir /s /q p:\subprojects\performance\build\santaTrackerAndroidBuild && (echo Directory removed) || (echo Directory not found) """
```

Comment:
❓ Shouldn't the smoke tests make a copy of this directory before running any build on it?

Reply:
Why can't the clean task delete read-only files? Shouldn't it be able to change the access to those files to writeable?